src/lib.rs

```diff
@@ -38,6 +38,43 @@ pub struct Decoder<R> {
 // internally), even though it's safe to send it across thread boundaries.
 unsafe impl<R: Send> Send for Decoder<R> {}
 
+/// A collection of pcm data decoded from a frame.
+///
+/// The data is stored in a channel interleaved fashion. You'll have to look at
+/// the associated [`FrameInfo`](FrameInfo) to determine its structured. It
+/// provides access to the underlying data by dereferencing to `&[i16]`.
+///
+/// This is provided for use with
+/// [`Decoder::decode_frame_into`](Decoder::decode_frame_into) which allows for
+/// re-using the buffer used for decoding data.
+///
+/// ```rust
+/// let pcm = minimp3::Pcm::new();
+///
+/// assert_eq!(&pcm[..], &[]);
+/// ```
+#[derive(Debug)]
+pub struct Pcm {
```

Comment:
`PcmFrame` or `FrameSamples`, consistent with the naming of `FrameInfo`? Because it is supposed to store the samples decoded from a single MP3 frame.